### PR TITLE
Main navigation

### PR DESCRIPTION
**Pull request by vladbailescu**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix SITES-9057

Test URLs:
- Before: https://main--ralphlauren--hlxsites.hlx.page/
- After: https://sites-9057--ralphlauren--hlxsites.hlx.page/
- Main: https://main--ralphlauren--hlxsites.hlx.page/index

### CODE REVIEW

**Comment by Buuhuu** [comment]
@vladbailescu can we fix the layout shift by overlapping the blue top banner with the hero?

File: styles/styles.css [hunk 1]
```diff
@@ -17,6 +17,20 @@
   font-display: swap;
 }
 
+@font-face {
```
Comment (rofe) [inline styles/styles.css:20]: As a general rule, never load external fonts in styles.css (synchronous) as it hurts the lighthouse score. Use font fallbacks instead, and load the fonts in lazy-styles.css.

Check out this helper tool to help generate matching font fallbacks: https://github.com/adobe/helix-font-fallback-extension

**Review by rofe — CHANGES_REQUESTED** [review]
I think the sections need a bit more spacing.

**Comment by rofe** [comment]
> @vladbailescu can we fix the layout shift by overlapping the blue top banner with the hero?

Hint: you can observe/debug layout shifts by throttling the network to slow 3G in chrome dev tools.

File: styles/styles.css (hunk 1 above)
Comment (vladbailescu) [reply styles/styles.css:20]: @rofe not sure what you mean by "external fonts", the fonts are loaded from our site, not another one. Also, I would leave this to the final optimization phase, for now trying to figure to get the general look and feel across the whole page.

**Comment by vladbailescu** [comment]
@Buuhuu 6167904 I get a score of 97 and 0 CLS
image

**Review by Buuhuu — APPROVED** [review]

**Comment by rofe** [comment]
Awesome performance optimization work @vladbailescu!

Bear in mind that this is on `.hlx.page` (i.e. uncached) where we generally have a larger variance.

**Pull request merged**